lib/api_request/models/product_viewdata.dart

```diff
@@ -22,5 +23,6 @@ class ProductViewData {
     required this.category,
     required this.thumbnail,
     required this.images,
+    required this.quantity,
```

Comment:
esse caso aqui é manipulado no carrinho né?
se for assim poderia deixar dessa forma:
```suggestion
    this.quantity = 0,
```